scripts/west_commands/run_common.py

```diff
@@ -340,14 +347,16 @@ def dump_traceback():
 
 def dump_context(command, args, unknown_args):
     build_dir = get_build_dir(args, die_if_none=False)
+    runner_configs = []
     if build_dir is None:
         log.wrn('no --build-dir given or found; output will be limited')
         runner_config = None
```

Comment:
```suggestion
        runner_configs = []
```

Either do this and remove the declaration above, or remove this declaration, because currently `runner_config` is left as an unused variable.

Reply:
removed.